x/data/spec/02_state.md

```diff
@@ -1 +1,3 @@
 # State
+
+The `data` module uses a lookup table as an abstraction over Cosmos SDK's `KVStore` for storing content hashes that adhere to a canonical regen IRI scheme. The lookup table creates a short, unique binary identifier for the longer content hash using an efficient, non-cryptographic hash function.
```

Comment:
```suggestion
The `data` module uses a lookup table for storing content hashes that adhere to a canonical regen IRI scheme. The lookup table creates a short, unique binary identifier for the longer content hash using an efficient, non-cryptographic hash function.
```

i feel like the specifics of the storage aren't too important, especially as we migrate towards the ORM, just the general structure of how the data is stored should be good enough imo